01-ganga-with-cmake.md

```diff
@@ -82,57 +76,41 @@ $ make install
 ```
 
 This will create a folder called `InstallArea`. Ganga needs to 
-know about this folder a couple of others to use the local project. We will create [symbolic links][symlink] 
-to these folders in the place that Ganga looks.
-
-```shell
-$ mkdir -p ~/cmtuser/DaVinci_v40r1p3
-$ cd ~/cmtuser/DaVinci_v40r1p3
-$ ln -s ~/DaVinciDev_v40r1p3/cmt
-$ ln -s ~/DaVinciDev_v40r1p3/DaVinciDevSys
-$ ln -s ~/DaVinciDev_v40r1p3/InstallArea
-$ ls -la
-drwxr-xr-x.  2 username z5 2048 May 19 09:05 .
-drwxr-xr-x.  2 username z5 2048 May 19 09:05 ..
-lrwxr-xr-x.  1 username z5   58 May 19 09:05 cmt -> /afs/cern.ch/user/a/apearce/DaVinciDev_v40r1p3/cmt
-lrwxr-xr-x.  1 username z5   58 May 19 09:05 DaVinciDevSys -> /afs/cern.ch/user/a/apearce/DaVinciDev_v40r1p3/DaVinciDevSys
-lrwxr-xr-x.  1 username z5   58 May 19 09:05 InstallArea -> /afs/cern.ch/user/a/apearce/DaVinciDev_v40r1p3/InstallArea
-```
+know about the local project.
 
-You can see the link in the output of `ls -la`. For example, the arrow `->` 
-says that `InstallArea` is a link to our `InstallArea` created by `lb-dev`.
-
-Now we can proceed as usual, defining a job running DaVinci in Ganga.
+Now we can proceed as usual, defining a job running DaVinci in Ganga. But instead of
+using the built in version of our app (`DaVinci()`), we will use `GaudiExec()` to make a 
+custom application:
 
 ```shell
-$ lb-run Ganga v601r19 ganga
+$ lb-run Ganga v602r0 ganga
 ```
 
 ```
 j = Job(name='Local_DaVinci_Test')
-j.application = DaVinci(version='v40r1p3')
-j.applications.optsfile = [...]
+myApp = GaudiExec()
+myApp.directory = "/path/to/DaVinciDev_v41r2p1"
+myApp.options = ["/path/to/DaVinciDev_v41r2p1/myOptions.py"]
+j.application = myApp
 j.inputdata = [...]
 j.outputfiles = [LocalFile('*.root')]
 j.submit()
 ```
 
 You can then confirm that you have a `_2M` branch in your output ROOT file.
 
+> ## Running without gaudirun.py {.callout}
+> 
+> If you want to run a program directly, you can add `j.application.useGaudiRun = False`.
```

Comment:
Should we be more explicit about `.options` just being the list of arguments passed to the application? (At least, that's what I assume it means.) Or maybe explicitly say "look at `help(GaudiExec)`".